lib/services/rally/rally_hierarchical_requirement_resource.rb

```diff
@@ -64,7 +64,8 @@ def update(id, hrequirement, element_name, query_params = "")
     url = rally_secure_url_without_workspace(object_path(id, element_name)+query_params)
     payload_key = element_name
     if element_name == "UserStory"
-      payload_key = "HierarchicalRequierement"
+      hrequirement.delete(:Release) if has_children?(id, element_name)
```

Comment:
Yeah, I found the actual bug. The patch has been removed, and the bug fix appears to exhibit the functionality that we need